build.sbt

```diff
@@ -3,13 +3,18 @@ organization := "com.gu"
 
 sbtPlugin := true
 
-scalaVersion := "2.10.6"
+scalaVersion := "2.12.2"
+sbtVersion in Global := "1.0.0"
+crossSbtVersions := Seq("1.0.0", "0.13.16")
```

Comment:
Possibly not. I think there's a reasonable argument for just saying you have to jump to 1.0, but given it was so little effort, I'm minded to cross-publish until it becomes a pain.